kubernetes/net-tests/net-test_ds.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: net-test-ds
+  labels:
+    app: net-test-ds
+spec:
+  selector:
+    matchLabels:
+      app: net-test
+  template:
+    metadata:
+      labels:
+        app: net-test
+    spec:
+      containers:
+      - name: iperf-server
+        image: kitesproject/net-test
+        securityContext:
+          privileged: true
```

Comment:
Why do we need a privileged container? I think we can reduce the attack surface by enabling only the linux kernel capabilities that we need.

Reply:
For netsniff-ng, otherwise without these privileges doesn't work inside the pod.

Reply:
we need to run as root to generate and receive traffic using netsniff-ng and trafgen

Reply:
Do we need full root privileges? I think we can make it work with only the required capabilities. http://man7.org/linux/man-pages/man7/capabilities.7.html

Reply:
good point :-) but let's add an issue leave it for a future milestone if we want to use our system in a production environment... for taking the performance measurements of our synthetic testbeds having root privileges in these containers is OK

Reply:
Done, opened #111